core/model/work.py

```diff
@@ -59,12 +59,7 @@
 
 # Import related models when doing type checking
 if TYPE_CHECKING:
-    from core.model import (  # noqa: autoflake
-        CachedFeed,
-        CustomListEntry,
-        Library,
-        LicensePool,
-    )
+    from core.model import CustomListEntry, Library, LicensePool  # noqa: autoflake
```

Comment:
Minor: its unrelated to this PR, but I noticed it because of the changes on this line. We can remove the `# noqa: autoflake` comment here.